oat-sa/tao-test-runner-qti-fe
# Feature/nex 2170 enable scrollable passages

Related to: https://oat-sa.atlassian.net/browse/NEX-2170
Rely to: https://github.com/oat-sa/extension-tao-itemqti/pull/1570

How to test:
  check `config/taoTests/test_runner_plugin_registry.conf.php` 
```
   'taoQtiTest/runner/plugins/content/itemScrolling/itemScrolling' => array(
            'id' => 'itemScrolling',
            'module' => 'taoQtiTest/runner/plugins/content/itemScrolling/itemScrolling',
            'bundle' => 'taoQtiTest/loader/testPlugins.min',
            'position' => null,
            'name' => 'Enable effect from enable/disable scrolling option',
            'description' => 'Add behavior from enable/disable scrolling option',
            'category' => 'content',
            'active' => true,
            'tags' => array(
            )
        ),
```
- go to Item authoring tool
- Select or add textblock
- In properties column activate enable scroll checkbox and set some option.
- Run test preview and if content in text block is bigger then page height there should appear scroll

Also need:
